seedqreader.py

```diff
@@ -518,7 +519,7 @@ def list_available_cameras():
         available_cameras = []
         while True:
             cap = cv2.VideoCapture(index)
-            if not cap.isOpened():
+            if not cap.isOpened() and available_cameras:
                 if (index - int(available_cameras[-1])) > 2:
```

Comment:
if no available camera here we got an exception and the program stops, so we need to check it before trying to access the index -1

Reply:
I recall similar and failed to report it, maybe pyqt window didn't even start.  Plugged in usbcam, and solved.

Reply:
Yes, it won't start without a usb camera... but now with this small fix it will start with or without a usb_cam

Reply:
> if no available camera here we got an exception and the program stops, so we need to check it before trying to access the index -1

I got it but I think is better to check it there (L523)
```python
if available_cameras and (index - int(available_cameras[-1])) > 2:
```

if not, even if no camera, #0 will be listed